examples/with-headers/request.tf

```
@@ -0,0 +1,7 @@
+data "curl_request" "ipify" {
+  uri         = "https://api.ipify.org?format=json"
+  http_method = "GET"
+  headers = jsonencode({
+    Content-Type = "application/json"
+  })
+}
```

```suggestion
}

```

Please ensure files end with a single new line.